teodyseguin/gizra-homeassignment
# Feature/site components

**About**
This PR contains code for the `site_components` custom module.

The module currently provides one component named `person card`.

**Test Plan**
1. Run `composer install` in order to install `components` module.
2. Install the `site_components` custom module. Clear the cache afterwards.
3. Navigate to `/style-guide` page. From the page, you should see the list of person cards, as shown from the **demo** video below.

**Demo**
I have uploaded it to my personal dropbox account as the sample gif where too big to be uploaded here on github.

https://www.dropbox.com/s/zhvnig0udo13q2v/style-guide.gif?dl=0
